configureBuild.sh

```diff
@@ -61,6 +61,11 @@ parseCommandLineArgs() {
     setOpenJdkVersion "$1"
     setDockerVolumeSuffix "$1"
   fi
+
+  # This check is to set the JAVA_HOME to the --jdk-boot-dir parameter (if both the param exists and JAVA_HOME doesn't)
+  if [ -d "${BUILD_CONFIG[JDK_BOOT_DIR]}" ] && [ -n "${JAVA_HOME:+x}" ]; then
```

Comment:
Is there any advantage to `-n ${JAVA_HOME:-x}` over `-z ${JAVA_HOME}` which seems a lot clearer?

Reply:
`-z` seems to cause an `unbound variable` exception when used with `JAVA_HOME`

Reply:
Hmmm ok that's due to `set -u` earlier in the script :-\
On that basis I guess we'll let this through despite it being less obvious